diff -ruN artkp/sample/simple/src/main.cpp artkp_mod/sample/simple/src/main.cpp
--- artkp/sample/simple/src/main.cpp	2006-07-25 16:05:46.000000000 -0700
+++ artkp_mod/sample/simple/src/main.cpp	2010-10-28 14:32:57.310876811 -0700
@@ -46,7 +46,6 @@
 
 #include "ARToolKitPlus/TrackerSingleMarkerImpl.h"
 
-
 class MyLogger : public ARToolKitPlus::Logger
 {
     void artLog(const char* nStr)
diff -ruN artkp/src/librpp/rpp.cpp artkp_mod/src/librpp/rpp.cpp
--- artkp/src/librpp/rpp.cpp	2006-04-19 23:28:10.000000000 -0700
+++ artkp_mod/src/librpp/rpp.cpp	2010-10-28 14:32:57.310876811 -0700
@@ -50,6 +50,7 @@
 #include "rpp.h"
 #include "rpp_const.h"
 #include "rpp_vecmat.h"
+#include <string.h>
 
 namespace rpp {
 
diff -ruN artkp/src/librpp/rpp_vecmat.cpp artkp_mod/src/librpp/rpp_vecmat.cpp
--- artkp/src/librpp/rpp_vecmat.cpp	2006-04-19 23:28:10.000000000 -0700
+++ artkp_mod/src/librpp/rpp_vecmat.cpp	2010-10-28 14:32:57.310876811 -0700
@@ -44,7 +44,8 @@
 #include "rpp_vecmat.h"
 #include "math.h"
 #include "assert.h"
-
+#include <stdio.h>
+#include <malloc.h>
 
 namespace rpp {
 
diff -ruN artkp/src/TrackerImpl.cxx artkp_mod/src/TrackerImpl.cxx
--- artkp/src/TrackerImpl.cxx	2006-07-25 16:05:46.000000000 -0700
+++ artkp_mod/src/TrackerImpl.cxx	2010-10-28 14:32:57.310876811 -0700
@@ -446,7 +446,7 @@
 		return(false);
 	}
 
-	pCam->changeFrameSize(AR_TEMPL_TRACKER::screenWidth,AR_TEMPL_TRACKER::screenHeight);
+	pCam->changeFrameSize(nWidth,nHeight);
 
 	int i;
     for(i = 0; i < 4; i++ )
